### PR TITLE
SIP-367: Synthetix V3 Deployment to Arbitrum (Mainnet)

### PR DESCRIPTION
**Pull request by noisekit**

### CODE REVIEW

**Review by dbeal-eth — CHANGES_REQUESTED** [review]
feel like there is generally a lot of code duplication that isnt necessary here, wrt the collateral configurations, oracle setup. at the very least seems to be avoidable for tesnet/mainnet?

i understand if the code duplication is deemed unavoidable. some other comments if you could confirm

File: omnibus-arbitrum-mainnet.toml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        { marketId = "<%= imports.perpsFactory.settings.superMarketId %>", weightD18 = 98, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
+    ],
+]
+
+[invoke.CoreProxy_initOrUpgradeToken_DollarWifHat]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "initOrUpgradeToken"
+args = [
+    "0x555344546f6b656e000000000000000000000000000000000000000000000000",
```
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml:207]: Same as in v3 repo. It has be copied around all of the deployments.

File: omnibus-arbitrum-mainnet.toml [hunk 2]
```diff
@@ -0,0 +1,212 @@
+name = "synthetix-omnibus"
+version = "1"
+description = "Arbitrum deployment"
```
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml]: Updated to match mainnet

File: omnibus-arbitrum-mainnet.toml [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+defaultValue = "<%= parseEther(String(100_000)) %>"
+
+[provision.dai_mock_collateral]
+source = "mintable-token:1.8"
+options.name = "Fake DAI Coin"
+options.symbol = "fDAI"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+[provision.arb_mock_collateral]
```
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml]: Removed

File: omnibus-arbitrum-mainnet.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml]: Added readable version

File: omnibus-arbitrum-mainnet.toml [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+description = "Pyth settlement strategy config"
+
+[provision.pyth_erc7412_wrapper]
+source = "pyth-erc7412-wrapper:3.3.15"
+targetPreset = "<%= settings.target_preset %>"
+options.salt = "<%= settings.salt %>"
+options.pythAddress = "<%= settings.pyth_price_verification_address %>"
+
+[setting.owner]
+defaultValue = "0xEde8a407913A874Dd7e3d5B731AFcA135D30375E"
```
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml]: Updated to match mainnet

**Comment by noisekit** [comment]
Copy paste is intentional. Copy first, refactor and unify next. I will do unification separately.

File: omnibus-arbitrum-mainnet.toml (hunk 4 above)
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml]: FIX: `setPoolCollateralConfiguration` should be called as pool owner and not contract owner

**Review by dbeal-eth — APPROVED** [review]

**Comment by dbeal-eth** [comment]
confirmed that the core proxy address is as expected for cross chain possible in the future :+1:

**Pull request merged**